### PR TITLE
Create BLorient8821 + corrections to BLorient8820

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient8821 according to S. Strelcyns catalogue.

Eventually I came across some mistakes or shortcomings in the record of BLorient8820, that I want to correct in this branch.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient8821.xml [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper">Oriental paper</material>
+                                    <material key="paper">European paper</material><locus target="#I #III"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+196+2</measure>
+                                    <measure unit="page" type="blank" corresp="#2v #172v">2</measure>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient8821.xml:178]: I am not sure if corresp works here with the page references...I always used a separate locus element in these situations

**Review by thea-m — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient8821.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient8821.xml:178]: I employed the same method in the record for BLorient8824. If this is not correct, I should change it in both cases and probably also in other records.

File: LondonBritishLibrary/orient/BLorient8821.xml [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+                <language ident="la">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-02-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient8821.xml:288]: This also happened in about 5 or 6 other records for BL manuscripts that Emmel addressed.

**Review by eu-genia — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient8821.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient8821.xml:178]: Yes, I have corrected it for this manuscript. I will also correct for the other BL records, that I have created up to now. I will use another branch for that.

**Pull request merged**